liferay-objects/liferay-portal
# LPS-156881 Using lowercase 'a' as object field name is displayed as '%A'

Motivation
=======
This work solves the problem reported in the [LPS-156881](https://issues.liferay.com/browse/LPS-156881) ticket.
cc: @sontruongces

Proposed Solution
========
What we did to solve this problem.
1. Disable localize label for aui:option in object_definition/details.jsp
2. Check isLocalizeLabel for objectField and nonObjectField.

Steps to Verify
========
1. Start a clean 7.4 update 28 bundle (or master)
2. Create an Object by navigating to control panel > objects > "+"
3. Edit the newly created Object
4. From the Fields tab, add a new field with the label "a" and any
5. From the Details tab, navigate to Entry Display and select the Title Field dropdown menu.

- Expected Results: There will be a title field named "a".
- Actual Results: There is a title field named "%A".

You can also view this issue by following these additional steps after step 4.
1. From the Details tab, enable Show Widget and click Publish
2. Go to the Home Page and edit it.
3. Navigate to the widget in the sidebar and add the newly created object from the Object section by dragging it on the page. Then click Publish.
4. From the Object portlet, create a new Object with the field and click Save and then Cancel to view the Object on the Home Page.

- Expected Results: There is a table column label named "a".
- Actual Results: There is a table column label named "%A".
